src/main/resources/db/changelog/changes/03-create-users-table.yaml

```diff
@@ -21,7 +21,7 @@ databaseChangeLog:
                     unique: true
               - column:
                   name: password
-                  type: varchar(20)
+                  type: varchar(60)
```

Comment:
pls avoid updating existing migrations in future

while this is ok for local setup (you just clear Liquibase tables in your local DB), it is not ok for real environment

therefore don't get used to it